### DISCUSSION TITLE
ESP32 Cheap Yellow Display  Board  -  Wie wird es bei u8g2 angeschlossen ?

### DISCUSSION THREAD

**Discussion by Zurrmaxe** [olikraus/u8g2#2449] [General]
Hallo zusammen

Ich habe folgendes ESP32-Display-Board und möcchte es gerne mit der u8g2 Libary betreiben.
Hier gibts es weitere Infos dazu:
https://randomnerdtutorials.com/cheap-yellow-display-esp32-2432s028r/#config-file-windows-pc

Hat es schon jemand am laufen bekommen ?

Mit den dort angegeben Beispielen läuft das Board bei mir.
Aber ich welche Verdrahtung benötige ich für die u8g2-Libarys ?
Alle versuche bisher waren erfolglos. 

Die vorhandenen Config-Zeilen in den u8g2-Libarys für das ILI9341 passen irgendwie nicht.

mfg.
Alfred

**Comment by olikraus** [comment]
U8g2 unterstützt nur monochrome Displays. Soweit ich sehen konnte ist dein Display ein RGB Display, das nicht unterstützt wird.

**Comment by Zurrmaxe** [comment]
Ja es ist ein Farbdisplay. In den Demos zur Libary wurden u8g2 - ili9341 Einstellungen angeführt.Deswegen hatte ich gehofft, dass es nur an den richtigen Ports liegt.
-------- Ursprüngliche Nachricht --------Von: olikraus ***@***.***> Datum: 24.05.24  19:34  (GMT+01:00) An: olikraus/u8g2 ***@***.***> Cc: Alfred ***@***.***>, Author ***@***.***> Betreff: Re: [olikraus/u8g2] ESP32 Cheap Yellow Display  Board  -  Wie wird es bei u8g2 angeschlossen ? (Discussion #2449) 
U8g2 unterstützt nur monochrome Displays. Soweit ich sehen konnte ist dein Display ein RGB Display, das nicht unterstützt wird.

**Reply by olikraus** [reply]
Tja, von mir stammt die Info vermutlich nicht 🤪

**Comment by Zurrmaxe** [comment]
Sorry war mein Fehler mit der u8g2 Lib.Ich hatte es in der ucglib Demo Hello World gefunden.//Ucglib_ILI9341_18x240x320_SWSPI ucg(/*sclk=*/ 7, /*data=*/ 6, /*cd=*/ 5, /*cs=*/ 3, /*reset=*/ 4);//Ucglib_ILI9341_18x240x320_SWSPI ucg(/*sclk=*/ 13, /*data=*/ 11, /*cd=*/ 9, /*cs=*/ 10, /*reset=*/ 8);//Ucglib_ILI9341_18x240x320_HWSPI ucg(/*cd=*/ 9, /*cs=*/ 10, /*reset=*/ 8);//Ucglib_ILI9341_18x240x320_SWSPI ucg(/*sclk=*/ 4, /*data=*/ 3, /*cd=*/ 6, /*cs=*/ 7, /*reset=*/ 5);	/* Das klappte alles nicht.Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: olikraus ***@***.***> Datum: 24.05.24  23:40  (GMT+01:00) An: olikraus/u8g2 ***@***.***> Cc: Alfred ***@***.***>, Author ***@***.***> Betreff: Re: [olikraus/u8g2] ESP32 Cheap Yellow Display  Board  -  Wie wird es
  bei u8g2 angeschlossen ? (Discussion #2449) 
Tja, von mir stammt die Info vermutlich nicht 🤪
